include/ofi_mr.h

```diff
@@ -285,7 +285,7 @@ struct ofi_mr_cache {
 
 	struct ofi_rbmap		tree;
 	struct dlist_entry		lru_list;
-	struct dlist_entry		flush_list;
+	struct dlist_entry		deferred_list;
```

Comment:
I actually find the term flush more descriptive than deferred here.  Flush at least indicates that the entry is queued to be removed from the cache, and the name aligns with the API (ofi_mr_cache_flush), and common cache term.  Deferred doesn't indicate what action is being deferred - registration? deregistration?

Reply:
For many people, in the context of a cache, the term flush means to flush the LRU list. So many people are confused when they found there are both `flush_list` and `lru_list` inside MR cache. Moreover, deferred de-registration is a quite unique behavior, using a general term 'flush_list' for an unique behavior added to their confusion. In the past, I had to explain this point in several discussions, which motivated the change.

If the word 'deferred' is too ambiguous. maybe something like 'staled' or 'zombie'?